components/bounty/page-actions/view.tsx

```diff
@@ -0,0 +1,289 @@
+                isLoading={isExecuting}
+              >
+                <span>
+                  <Translation ns="bounty" label="actions.start-working.title"/>
+                </span>
+              </ContractButton>
+            </ReadOnlyButtonWrapper>
+        );
+      }
+    }
+  }
+
+  function renderCreatePullRequestButton() {
+    if (isCreatePr)
+      return (
+        <ReadOnlyButtonWrapper>
+          <ContractButton
+            className="read-only-button bounty-outline-button"
+            onClick={() => handleShowPRModal(true)}
+            disabled={!currentUser?.login || !isWalletConnected}
+          >
+            <Translation ns="pull-request" label="actions.create.title"/>
+          </ContractButton>
+        </ReadOnlyButtonWrapper>
+      );
+  }
+
+  function renderUpdateAmountButton() {
+    if (isWalletConnected && isBountyOpen && isBountyOwner && isBountyInDraft && !isFundingRequest && !isEditIssue)
+      return (
+        <ReadOnlyButtonWrapper>
+          <ContractButton
+            className="read-only-button bounty-outline-button me-1"
+            onClick={() => setShowUpdateAmount(true)}
+          >
+            <Translation ns="bounty" label="actions.update-amount"/>
+          </ContractButton>
+        </ReadOnlyButtonWrapper>
+      );
+  }
+
+  function renderCreateProposalButton() {
+    if (isCreateProposal)
+      return (
+        <ReadOnlyButtonWrapper>
+          <ContractButton
+            className="read-only-button bounty-outline-button"
+            onClick={() => handleShowPRModal(true)}
+            disabled={!currentUser?.login || !isWalletConnected}
+          >
+            <Translation ns="proposal" label="actions.create.title"/>
+          </ContractButton>
+        </ReadOnlyButtonWrapper>
+      );
+  }
+
+  function renderEditButton() {
+    if (isWalletConnected && isBountyInDraft && isBountyOwner)
+      return (
+        <ReadOnlyButtonWrapper>
+          <ContractButton
+            className="read-only-button bounty-outline-button me-1"
+            onClick={handleEditIssue}
+          >
+            <EditIcon className="me-1"/>
+            <Translation ns="bounty" label="actions.edit-bounty" />
+          </ContractButton>
+        </ReadOnlyButtonWrapper>
+      );
+  }
+
+  function renderTableAndMobileButton() {
+    const actions: Action[] = []
+
+    if (isCreatePr)
+      actions.push({
+        label: "Pull Request",
+        onClick: () => handleShowPRModal(true),
+      });
+      
+    if (isCreateProposal)
+      actions.push({
+        label: "Proposal",
+        onClick: () => setShowPRProposal(true),
+      });
+
+    if (!isGithubConnected && isWalletConnected)
+      return <ConnectGithub size="lg" />;
+
+    if (isCreatePr || isCreateProposal)
+      return (
+          <MultiActionButton
+            label="Create"
+            className="col-12"
+            actions={actions}
+          />
+      );
+
+    return renderStartWorkingButton();
```

Comment:
don't hide renders within renders. Some of these renders can also be `views` inside this folder [same applies for other function/components on other files]